modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -18,9 +18,9 @@ Install the rtfctl utility to manage Runtime Fabrics locally, with support for t
 You may need root privileges to a controller node running Runtime Fabric to follow this procedure.
 
 . Download the https://runtime-fabric.s3.amazonaws.com/rtfctl/rtfctl-0.2.32[latest version] of the `rtfctl` utility.
-. Transfer the `rtfctl` binary to a controller node under the `/usr/local/bin` folder.
+. Transfer the `rtfctl` binary to a controller node under the `/local/usr/bin` folder.
```

Comment:
I think this should still be /usr/local/bin

Reply:
Got it - thanks!